examples/3d/irradiance_volumes.rs

```diff
@@ -132,8 +132,8 @@ struct VoxelVisualizationExtension {
 
 #[derive(ShaderType, Debug, Clone)]
 struct VoxelVisualizationIrradianceVolumeInfo {
-    transform: Mat4,
-    inverse_transform: Mat4,
+    world_from_voxel: Mat4,
+    voxel_from_world: Mat4,
```

Comment:
Same here with voice not being a space, but now I have seen the light from world and so on and I like that. I’m open for debate.

Reply:
I’m ok with this now.